chain/src/types.rs

```diff
@@ -57,6 +57,8 @@ pub enum Error {
 	InvalidBlockTime,
 	/// Block height is invalid (not previous + 1)
 	InvalidBlockHeight,
+    /// coinbase can only be spent after it has matured (n blocks)
+    ImmatureCoinbase,
```

Comment:
rustfmt? :-)

Reply:
Hmm - soft tabs or hard tabs?
Got my editor to show invisibles and it's hard tabs right?
(except the bitflags up top had spaces in them, which confused my editor...)